react-native-aztec/ios/Cartfile

```diff
@@ -1 +1 @@
-github "wordpress-mobile/AztecEditor-iOS" "1.10.1"
+github "wordpress-mobile/AztecEditor-iOS" "81e1e1bb1cb209004d66bba75338595cc9aab147"
```

Comment:
You can now update this to a release tag

Reply:
Just did! 😁